opensrp-chw/src/main/java/org/smartregister/chw/actionhelper/ECDAction.java

```diff
@@ -64,13 +64,13 @@ private String getTranslatedValue(String nativeResult) {
         if (StringUtils.isBlank(develop_warning_signs))
             return BaseAncHomeVisitAction.Status.PENDING;
 
-        if (
-                "No".equalsIgnoreCase(develop_warning_signs) ||
-                        "No".equalsIgnoreCase(stim_skills) ||
-                        (StringUtils.isNotBlank(early_learning) && "No".equalsIgnoreCase(early_learning))
+        if(
+                "No".equalsIgnoreCase(develop_warning_signs)
+                        && "Yes".equalsIgnoreCase(stim_skills)
```

Comment:
isn't it's checking with language specific?
you may using this method
Utils.getYesNoAsLanguageSpecific(context,stim_skills).equalsIgnoreCase(context.getString(R.string.yes))

Reply:
as the result comes always english so this checking is ok